workflow/common/common.go

```diff
@@ -159,6 +159,9 @@ const (
 	// EnvAgentPatchRate is the rate that the Argo Agent will patch the Workflow TaskSet
 	EnvAgentPatchRate = "ARGO_AGENT_PATCH_RATE"
 
+	// EnvExecGCWorkers is the number of artifact GC workers for argoexec
+	EnvExecGCWorkers = "ARGO_EXEC_GC_WORKERS"
+
```

Comment:
I don't think there's a code path that will apply this to the ArtifactGCPod. Somebody would have to enable environment variable specification as part of the `ArtifactGC` spec. Were you planning on adding that?

Reply:
nice!